openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -689,3 +693,56 @@ def delete(self, request, usage_key_str, file_path):
         except ValueError:
             raise ValidationError("Invalid file path")  # lint-amnesty, pylint: disable=raise-missing-from
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+@view_auth_classes()
+class LibraryImportTaskViewSet(ViewSet):
+    """
+    Import blocks from Courseware through modulestore.
+    """
+
+    @convert_exceptions
+    def list(self, request, lib_key_str):
+        """
+        List all import tasks for this library.
+        """
+        library_key = LibraryLocatorV2.from_string(lib_key_str)
+        api.require_permission_for_library_key(library_key, request.user, permissions.CAN_EDIT_THIS_CONTENT_LIBRARY)
+        queryset = api.ContentLibrary.objects.get_by_key(library_key).import_tasks
+        return Response(ContentLibraryBlockImportTaskSerializer(queryset, many=True).data)
+
+    @convert_exceptions
+    def create(self, request, lib_key_str):
```

Comment:
I am assuming now that you are performing tests with the devstack.

This is because the devstack [is configured to not send tasks to the workers and run them synchronously](https://github.com/edx/edx-platform/blob/master/cms/envs/devstack.py#L81). I'll update the testing instructions with information on how to bypass that.